src/sayt/index.ts

```diff
@@ -15,6 +15,8 @@ class Sayt {
   };
 
   init() {
+    // initialize as active to initialize child component
+    this.on('mount', this.setInactive);
```

Comment:
can actually be done using the lifecycle sugar method `onMount()` instead of attaching a new listener